modules/persistent-storage-fibre-provisioning.adoc

```diff
@@ -32,11 +32,13 @@ spec:
   accessModes:
     - ReadWriteOnce
   fc:
-    targetWWNs: ['500a0981891b8dc5', '500a0981991b8dc5'] <1>
-    lun: 2
+    wwids: [scsi-3600508b400105e210000900000490000] <1>
+    targetWWNs: ['500a0981891b8dc5', '500a0981991b8dc5'] <2>
+    lun: 2 <2>
```

Comment:
Perhaps this should be a <3>, followed by an explanation about what this is about.